helm/fluent-logshipping-app/templates/configmap.yaml

```diff
@@ -447,7 +448,7 @@ data:
         s3_key_format                {{ printf "/ssh%s" .Values.outputs.aws.S3.s3_object_key_format }}
         {{- end }}
         s3_key_format_tag_delimiters .-_
-        store_dir                    ${storage_path}/S3
+        store_dir                    ${storage_path}/S3/ssh/sshd
```

Comment:
Changing this otherwise the output will send all the logs in there to the same folder in the bucket :(